acceptance/bundle/paths/relative_path_translation/script

```diff
@@ -0,0 +1,12 @@
+errcode trace $CLI bundle validate -t default -o json | jq '.resources.jobs.job' > output.default.json
+errcode trace $CLI bundle validate -t override -o json | jq '.resources.jobs.job' > output.override.json
```

Comment:
The original test only looked at tasks, why not fetch them here and get rid of the jq assertion below and rely on golden file?

Reply:
Done.